Lib/test/test_strptime.py

```diff
@@ -135,6 +135,8 @@ def test_pattern_escaping(self):
                       "%s does not have re characters escaped properly" %
                       pattern_string)
 
+    @unittest.skipIf(sys.platform == 'win32' and locale.getdefaultlocale()[1] == 'cp65001',
```

Comment:
Can you add a comment referencing bpo-36511 here, so people in the future will know when it is safe to stop skipping this?

It might even be worth opening a separate BPO issue for tracking this specific thing, because issue 36511 seems like more of an omnibus issue.

@zooba You probably know more about the lifetime of Windows support here - what is the best way to communicate that it will be safe to remove this `skipIf`?

Reply:
Agreed, a specific issue for this particular failure would be good. There was one already for the same thing (to do with passing `de_DE` instead of `de-DE` IIRC, which triggered the experimental/broken UTF-8 support in the UCRT) that might be okay to reuse.

I don't know how best to communicate a future time for this, especially since the UCRT may get updated down-level (in earlier releases) and certain fixes may release out of band. Probably best to just leave a comment with the current UCRT version (@paulmon?) and a reminder to verify as new updates are made.

Reply:
@zooba, Is this the same issue? https://bugs.python.org/issue36792

Reply:
I created a new issue: https://bugs.python.org/issue37552 and added comments.